build-tools/packages/build-cli/docs/generate.md

```diff
@@ -3,12 +3,65 @@
 
 Generate commands are used to create/update code, docs, readmes, etc.
 
+* [`flub generate asserts`](#flub-generate-asserts)
 * [`flub generate buildVersion`](#flub-generate-buildversion)
 * [`flub generate bundleStats`](#flub-generate-bundlestats)
 * [`flub generate changelog`](#flub-generate-changelog)
 * [`flub generate changeset`](#flub-generate-changeset)
 * [`flub generate upcoming`](#flub-generate-upcoming)
 
+## `flub generate asserts`
```

Comment:
Maybe should be `generate assertTags`?

Reply:
maybe if theres more types of asserts to generate, we can use the `assert:tags` syntax and move it to a subfolder within assert? Might be wrong, just a thought.

Reply:
Since we use "tag" as the verb when we talk about this process, I'm going with tag:asserts, even though it's unlikely we'll have other things under "tag."

Reply:
I went back to "generate assertTags"... Using "tag" just for this command didn't sit well with me. Naming things is hard.